scss/main.scss

```diff
@@ -886,6 +886,10 @@ h1,h2,h3 {
   }
 }
 
+.card3Row {
```

Comment:
Might be worth refactoring ".card3" to something like ".image-left-card" so that we can re-use the first and third class since they should behave the same. That ever there is a design with more than 3 cards we don't have to keep adding new classes.